packages/auto-label/README.md

```diff
@@ -106,7 +106,7 @@ path:
 To define how certain directories should be labeled (Note: labels will respect the deepest path configuration):
 ```yaml
 # In .github/auto-label.yaml
-language:
+path:
```

Comment:
was this just a typo, or would we need to update config in a few places?

Reply:
This was just a typo in the readme. It wouldn't have worked in the config. And the configs I'm aware of do say "path" :)